src/include/sof/audio/mfcc/mfcc_comp.h

```diff
@@ -125,8 +125,8 @@ struct mfcc_state {
 	int low_freq;
 	int high_freq;
 	int sample_rate;
-	int waiting_fill:1; /**< booleans */
-	int prev_samples_valid:1;
+	bool waiting_fill:1; /**< booleans */
+	bool prev_samples_valid:1;
```

Comment:
there's also #9385 that's fixing this bug, and it was submitted a bit earlier, but this commit fixes an additional error. Same comment as there - let's just make both these fields `bool` without bit-picking.

Reply:
@cujomalainey I think I will drop this from PR and leave it for your PR to address.